Add a gist of the difference between code-server and Coder (#4419)
cdr/code-server: docs/README.md

- Use cloud servers to speed up tests, compilations, downloads, and more
- Preserve battery life when you're on the go; all intensive tasks run on your
  server
  

> | 🔔 code-server is the free and libre single-user variant of [Coder](https://coder.com/), our enterprise developer workspace platform. For more information, visit [Coder.com](https://coder.com/docs/comparison)

Reviewer: What is the `|` for? Do we need that?

Author: Yeah, idk but I think it helps making the box.

Author: Wait, I understood! In Markdown, you need the `|` to make a table, right? We made a 1x1 table aka. the banner that has the alert! In Markdown, you make tables like this:
```markdown
|      |      |      |
| ---- | ---- | ---- |
|      |      |      |
|      |      |      |
|      |      |      |
```

How it looks:
|      |      |      |
| ---- | ---- | ---- |
|      |      |      |
|      |      |      |
|      |      |      |

So, we used this:
```markdown
| Hello
| ---
```
And it looks like this:
| Hello
| ---

Do you get it?

Reviewer: OH! That makes sense. Let's keep it then 👍  Thanks for the explanation and demo :)

> | 🔔 code-server is the free and libre single-user variant of [Coder](https://coder.com/), our enterprise developer workspace platform. For more information, visit [Coder.com](https://coder.com/docs/comparison)

Reviewer: On second thought, can you change this to this?

```suggestion
| 🔔 code-server is a free browser-based IDE while [Coder](https://coder.com/), is our enterprise developer workspace platform. For more information, visit [Coder.com](https://coder.com/docs/comparison)
```
